packages/sdks/web-component/rollup.config.app.serve.mjs

```diff
@@ -4,5 +4,12 @@ import conf from './rollup.config.app.mjs';
 // If TS type errors remove bs-recipes references in package-lock.json due to bad global React typings
 export default {
   ...conf,
-  plugins: [...conf.plugins, browsersync({ server: 'build' })],
+  plugins: [
+    ...conf.plugins,
+    browsersync({
+      server: 'build',
+      watch: true,
+      reloadOnRestart: true,
```

Comment:
Do we need it?

Reply:
it made my life easier testing and debugging, used only in development, has no impact on published package